.github/workflows/dotnet.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: Setup Dot Net Version
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 6.0.100
+        dotnet-version: 7.0.0-preview.1
```

Comment:
This should be a separate PR.

Reply:
Reverted to 6.0.100 . I will do that in a separate PR